commander/tcs.py

```diff
@@ -72,10 +76,68 @@ async def auxtel_docstrings(request):
         docstrings = {m: getattr(atcs_client, m).__doc__ for m in methods}
         return web.json_response((docstrings), status=200,)
 
+    async def connect_to_mtcs_intance():
+        global mtcs_client
+        try:
+            mtcs_client = MTCS()
+            await mtcs_client.start_task
+        except Exception:
+            mtcs_client = None
+
+    connect_to_mtcs_intance()
+
+    async def maintel_command(request):
+        global mtcs_client
+        if not mtcs_client:
+            await connect_to_mtcs_intance()
+        if not mtcs_client:
+            return unavailable_mtcs_client()
+
+        req = await request.json()
+
+        command_name = req["command_name"]
+        params = req["params"]
+
+        try:
+            command = getattr(mtcs_client, command_name)
+        except Exception:
+            resp = {"ack": f"Invalid command"}
+            return web.json_response(resp, status=400,)
+        try:
+            result = await command(**params)
+            return web.json_response({"ack": str(result)}, status=200,)
+        except Exception as e:
+            resp = {"ack": f"Error running command {command_name}: {e}"}
+            return web.json_response(resp, status=400,)
+
+    async def maintel_docstrings(request):
+        global mtcs_client
+        if not mtcs_client:
+            await connect_to_mtcs_intance()
+        if not mtcs_client:
+            return unavailable_mtcs_client()
+
+        methods = [
+            func
+            for func in dir(mtcs_client)
+            if callable(getattr(mtcs_client, func)) and not func.startswith("__")
+        ]
+        docstrings = {m: getattr(mtcs_client, m).__doc__ for m in methods}
+        return web.json_response((docstrings), status=200,)
+
+    def unavailable_mtcs_client():
```

Comment:
I would move this method between `connect_to_mtcs_intance` and `maintel_command`, near line 88